docs/user/aws/install_upi.md

```diff
@@ -65,9 +65,30 @@ data["spec"]["mastersSchedulable"] = False;
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 ```
 
-### Remove DNS Zones
+### Adjust DNS Zones
 
-If you don't want [the ingress operator][ingress-operator] to create DNS records on your behalf, remove the `privateZone` and `publicZone` sections from the DNS configuration:
+[The ingress operator][ingress-operator] is able to manage DNS records on your behalf.
+Depending on whether you want operator-managed DNS or user-managed DNS, you can choose to [identify the internal DNS zone](#identify-the-internal-dns-zone) or [remove DNS zones](#remove-dns-zones) from the DNS configuration.
+
+#### Identify the internal DNS zone
+
+If you want [the ingress operator][ingress-operator] to manage DNS records on your behalf, adjust the `privateZone` section in the DNS configuration to identify the zone it should use.
```

Comment:
AFAIK ingress-operator adds the dns records to for `clusterdomain` only. Can it handle any other?

Reply:
> AFAIK ingress-operator adds the dns records to for clusterdomain only.

Ingress is filling `DNSRecord` requests.  In most cases now this is just going to be the `default-wildcard` record, and I dropped an example YAML dump of that into the 14e06912a31a commit message for the curious ;).  But folks pushing new `DNSRecord`s will probably get them filled, because I don't think the ingress operator has (or should have) guards about which DNSRecords it should or should not fill.  If you're authorized to push a `DNSRecord` where ingress is looking, you should get it filled.

Reply:
But in case where the private dns zone id is for `not-clusterdomain.` and DNSRecord created by default would fail right.

Reply:
> But in case where the private dns zone id is for `not-clusterdomain`. then the DNSRecord created by default would fail right.

Ah, probably.  Although it depends on the zone domains.  So you could have an `example.com` zone, and Ingress could create an `a.example.com` record even if your cluster was `c.d.e.example.com` (I think ;).  But yeah, I expect it would fail to create `example.edu`.  Does that matter?  We aren't telling them to create `DNSRecord`s here, we're just telling them how to feed zones to the ingress operator, which can have its own docs about what it can and cannot support.